Libplanet.Tests/Blockchain/BlockChainTest.Internals.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Libplanet.Crypto;
+using Libplanet.Tests.Common.Action;
+using Libplanet.Tx;
+using Serilog;
+using Xunit;
+
+namespace Libplanet.Tests.Blockchain
+{
+    public partial class BlockChainTest
+    {
+        [Fact]
+        public void ListStagedTransactions()
+        {
+            Transaction<DumbAction> MkTx(PrivateKey key, long nonce, DateTimeOffset? ts = null) =>
+                Transaction<DumbAction>.Create(
+                    nonce,
+                    key,
+                    _blockChain.Genesis.Hash,
+                    new DumbAction[0],
+                    null,
+                    ts ?? DateTimeOffset.UtcNow
+                );
+
+            PrivateKey a = new PrivateKey();
+            PrivateKey b = new PrivateKey();
+            PrivateKey c = new PrivateKey();
+            PrivateKey d = new PrivateKey();
+            PrivateKey e = new PrivateKey();
+
+            // 우리가 인위적으로 만들어야 하는 corner cases
+            // A. 정상 케이스 (3 txs: 0, 1, 2)
+            // B. 순서가 뒤집힌 nonce ( 2 tsx: 1, 0)
+            // C. 작은 nonce가 나중 timestamp (2 txs: 0 (later), 1)
+            // D. 이가 빠진 nonce (3 txs: 0, 1, 3)
+            // E. nonce가 겹치는 경우 (4 txs: 0, 1, 1, 2)
```

Comment:
```suggestion
            // Corner cases
            // A. Formal case (3 txs: 0, 1, 2)
            // B. Reverse-ordered nonce ( 2 tsx: 1, 0)
            // C. Smaller nonce has later timestamp (2 txs: 0 (later), 1)
            // D. Skipped nonce (3 txs: 0, 1, 3)
            // E. Duplicated nonce (4 txs: 0, 1, 1, 2)
```

Reply:
I am going to submit a separate equivalent patch to the 0.10-maintenance branch, and your suggestion will be included there.